archiver.go

```diff
@@ -103,6 +103,8 @@ type Extractor interface {
 // File provides methods for accessing information about
 // or contents of a file within an archive.
 type File struct {
+	FullPath string
```

Comment:
Would be useful to provide a comment here explaining what the field should contain.